feature/aft/aft_base/otg_tests/afts_base/README.md

```diff
@@ -61,7 +55,7 @@ Bring down the link between ATE:port2 and DUT:port2 using OTG api.
 
 *   BGP routes advertised from ATE:port1,port2 must have 1 nexthop.
 *   IS-IS routes advertised from ATE:port3 must have one next hop.
-*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+*   BGP routes advertised from ATE:port3 must have indirect next hop.
```

Comment:
Please be more specific about what this means -- you mean that they must recurse on to some other form of route. (You can do this by implementing my proposed rework of this README.)

Reply:
it must be ip of the neighbor, will add that details